CHANGELOG.md

```diff
@@ -3,8 +3,80 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.9.0
+_Apr 14, 2022_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Add support for React 18 (#4332) @m4theushw
+
+- 🎁 Add support for [Column spanning](https://mui.com/x/react-data-grid/columns/#column-spanning) (#4020) @cherniavskii
+
+  <img src="https://user-images.githubusercontent.com/13808724/162926746-93bcb180-3c9d-4eb9-afc7-c3908a5c6406.png" width="788">
+
+- 📚 New standalone documentation for MUI X (#4313) @siriwatknp
+
+  Each MUI product now have its own documentation.
+  More information on our [blog post](https://mui.com/blog/docs-restructure-2022/).
+
+- 🌍 Add Hungarian (hu-HU) locale (#4458) @x22tri
+
+- 🐞 Bug fixes
+
+### `@mui/x-data-grid@v5.9.0` / `@mui/x-data-grid-pro@v5.9.0`
+
+#### Changes
+
+- [DataGrid] Add indexes relative to the filtered rows and the current page to the `getRowClassName` and `getRowSpacing` props (#3882) @flaviendelangle
+- [DataGrid] Add React 18 to peer dependencies (#4332) @m4theushw
+- [DataGrid] Apply filtering before sorting (#4359) @flaviendelangle
+- [DataGrid] Column spanning (#4020) @cherniavskii
+- [DataGrid] Enable using non-native Select in filter panel (#4361) @kyeongsoosoo
+- [DataGrid] Fix `api` prop leaking to DOM (#4384) @m4theushw
+- [DataGrid] Fix column dimensions import/export with flex and resizing (#4311) @flaviendelangle
+- [DataGrid] Fix inconsistent overlay when changing the `loading` prop (#4334) @m4theushw
+- [DataGrid] Fix scrollbar grabbing issue in Safari (#4405) @cherniavskii
+- [DataGrid] `GridCellParams.formattedValue` should be nullable (#4376) @flaviendelangle
+- [DataGrid] Prevent column header scroll (#4280) @m4theushw
+- [DataGridPro] Fix toggling detail panel using keyboard (#4409) @cherniavskii
+- [l10n] Add Hungarian (hu-HU) locale (#4458) @x22tri
+
+### `@mui/x-date-pickers@5.0.0-alpha.1` / `@mui/x-date-pickers-pro@5.0.0-alpha.1`
+
+##### Changes
```

Comment:
What is the goal of the `Changes` title ?
The core don't have it

Reply:
I think it's there to separate highlights and other changes. But it seems useless when there's no highlights

Reply:
But now that we have highlights outside the packages section, I think we can just get rid of this header

Reply:
It made sense when there was breaking changes.